solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/UIViewController+ExtensionsTests.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIViewController+ExtensionsTests.swift
+//  FinanceAppTests
+//
+//  Created by Gabriel de Castro Chaves on 23/08/22.
+//
+
+import XCTest
+@testable import FinanceApp
+
+class DummyViewController: UIViewController {
```

Comment:
[suggestion] uma boa pratica e sempre colocar `final`:
```suggestion
final class DummyViewController: UIViewController {
```